curiouslearning/FeedTheMonsterJS
# fix: CR-166 tigragna language build

# Changes
- tigragna/audios
-ftm_tigragna.json

# How to test
- Check tigragna language

Ref: [CR-166](https://curiouslearning.atlassian.net/jira/software/projects/CR/boards/7?selectedIssue=CR-166)


[CR-166]: https://curiouslearning.atlassian.net/browse/CR-166?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
